src/libstore/worker-protocol.cc

```diff
@@ -31,14 +31,14 @@ std::optional<TrustedFlag> WorkerProto::Serialise<std::optional<TrustedFlag>>::r
 void WorkerProto::Serialise<std::optional<TrustedFlag>>::write(const Store & store, WorkerProto::WriteConn conn, const std::optional<TrustedFlag> & optTrusted)
 {
     if (!optTrusted)
-        conn.to << (uint8_t)0;
+        conn.to << uint8_t(0);
```

Comment:
How do you feel about parentheses vs curly braces for this?

Reply:
Use parentheses if you accept implicit conversion, and use curly braces if you want strict type matching. Generally prefer strict and defensive handling of types.

in this case, i want to convert "whatever integer to uint8_t", so it's parentheses.

Reply:
I would think 

- curly braces work in this case
- curly braces should be preferred basically any time they can be used

If it's a literal it doesn't feel like *convert* whatever integer, it should be "this integer is the right type from the get-go" right?